mars/dataframe/reduction/tests/test_reduction_execute.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+import numpy as np
+
+from mars.tests.core import TestBase
+from mars.dataframe.datasource.series import from_pandas as from_pandas_series
+from mars.dataframe.datasource.dataframe import from_pandas as from_pandas_df
+
+
+class Test(TestBase):
+    def testSeriesSum(self):
```

Comment:
`numeric_only` and `min_count` is not tested.

Reply:
`numeric_only` only for `DataFrame`.